portal/portal-sprint-103.md

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
Why is this file in this PR?
If there are any changes, they should be applied to this file in the 103 release note branch & PR.

Reply:
I honestly don't know how i managed to do that :D